server/connectors/notion_connector/client.py

```
@@ -0,0 +1,1032 @@
+                    feature,
+                )
+                return {"error": f"{feature}_not_available"}
+            # Some endpoints return 400 when the API version is too old or the
+            # feature is not available for this workspace/plan.  Treat these
+            # the same as 404 for optional-feature guards.
+            if status == 400:
+                body = {}
+                try:
+                    body = exc.response.json() if exc.response is not None else {}
+                except Exception:
+                    pass  # Malformed JSON in error response — use empty body
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)